cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -185,12 +192,29 @@ func TestValidateMasterConfiguration(t *testing.T) {
 		{&kubeadm.MasterConfiguration{
 			AuthorizationModes: []string{"RBAC"},
 			Networking: kubeadm.Networking{
+				ServiceSubnet: "2001:db8::1/98",
```

Comment:
Ditto here. Would be best served by a name field, IMHO, where you could indicate things like "missing token".